nupic/research/frameworks/continuous_learning/multihead/subdataset.py

```diff
@@ -36,7 +36,7 @@
 from torchvision.datasets import MNIST
 
 
-def get_datasets(dataset_name, scenario, download=False):
+def get_datasets(dataset_name, scenario, download=True):
```

Comment:
I originally changed this since the code would break since the unit tests required the data to run, but since we're going to use a FakeDataLoader, I will change it back to `download=False`.